cmake/extensions.cmake

```diff
@@ -1937,10 +1944,26 @@ Relative paths are only allowed with `-D${ARGV1}=<path>`")
   endif()
 
   if(FILE_CONF_FILES)
-    set(FILENAMES ${BOARD})
+    if(DEFINED FILE_BOARD_REVISION AND NOT FILE_BOARD)
+        message(FATAL_ERROR
+	  "zephyr_file(${ARGV0} <path> BOARD_REVISION ${FILE_BOARD_REVISION} ...)"
+	  " given without BOARD argument, please specify BOARD"
+	)
```

Comment:
This indentation is misleading.

Reply:
Thanks for spotting.
A couple of stray tabs that has now been converted to spaces.